python/gcp_client/setup.py

```diff
@@ -14,7 +14,7 @@
 SUBPACKAGE_SLUG = f"{NAMESPACE_PACKAGE_NAME}.{SUBPACKAGE_NAME}"
 
 # Subpackage version
-VERSION = "2.3.4"
+VERSION = "2.4.0"
```

Comment:
Given the above changes in canonical dataframe output and a few minor changes to the api, I think this really "should be" a major release bump. But if you disagree, id love to hear your thoughts!

Reply:
I think you're right